spec/reset_memo_wise_spec.rb

```diff
@@ -388,6 +388,48 @@
           expect(instance2.no_args_counter).to eq(1)
         end
       end
+
+      context "when method name is the same as a memoized class method" do
```

Comment:
I think it makes sense to either leave a comment explaining this decision or to add similar tests to `preset_memo_wise_spec`, otherwise they're asymmetrical without a documented explanation for why.